aea/helpers/pipe.py

```diff
@@ -511,8 +511,13 @@ def __init__(  # pylint: disable=unused-argument
         """
         self.logger = logger
         self._loop = loop
-
-        self._port = int(in_path)
+        if len(in_path.split(":")) == 1:
+            self._port = int(in_path)
```

Comment:
(minor) consider doing some validation and raise proper message error in case it cannot be decoded. You may use `RegexConstrainedString` for that